src/providers/MeetingProvider/docs/MeetingManager.stories.mdx

```diff
@@ -51,21 +28,38 @@ import { MeetingManagerConfig } from 'amazon-chime-sdk-component-library-react';
 (data: MeetingJoinData) => Promise<void>
 
 interface MeetingJoinData {
```

Comment:
Is there any use case we need this `MeetingJoinData`? From my opinion, if we only use it in `join()` API, then `MeetingJoinData` is an unnecessary abstraction. And I think it's better to have `meetingSessionConfiguration` and `options` as parameters in `join()` directly.

Reply:
Done